src/components/plots/barPlot.ts

```diff
@@ -171,12 +172,14 @@ export module Plots {
       }
     }
 
-    protected _setupDatasetNodes(dataset: Dataset) {
-      super._setupDatasetNodes(dataset);
+    protected _createNodesForDataset(dataset: Dataset) {
+      var drawer = super._createNodesForDataset(dataset);
+      drawer._getRenderArea().classed(Bar._BAR_AREA_CLASS, true);
```

Comment:
Shall we open an issue for this? I don't like this endpoint. [jQuery](https://api.jquery.com/addclass/) [is](https://api.jquery.com/hasclass/) [nicer](https://api.jquery.com/removeclass/)